packages/perseus/src/widgets/__tests__/__snapshots__/video_test.jsx.snap

```diff
@@ -63,7 +63,7 @@ exports[`video widget should snapshot on mobile: first mobile render 1`] = `
                   class="default_xu2jcg-o_O-inlineStyles_sc01dy"
                 />
                 <a
-                  class="shared_ken7mo-o_O-resting_nelfll visited-no-recolor"
+                  class="shared_htyo3s-o_O-resting_nelfll visited-no-recolor"
```

Comment:
There was a minor functional changes include in the new `wonder-blocks-link` package so this is expected.  This was necessary to move forward with `wonder-blocks-link` in webapp.

Reply:
cc @nishasy